test/integration/server-runas-root/inspec/server_runas_root_spec.rb

```diff
@@ -5,6 +5,12 @@
   title 'Verify Enterprise Splunk server installation'
   only_if { os.linux? }
 
+  describe 'chef-splunk::server should run as "root" user' do
+    describe command('ps aux | grep "splunkd --under-systemd" | head -1 | awk \'{print $1}\'') do
```

Comment:
Instead of using `ps` to check on processes, please use the [processes](https://docs.chef.io/inspec/resources/processes/) resource in Inspec. Please fix this globally as there are several places this has been done.